samples/cellular/nrf_provisioning/sample.yaml

```diff
@@ -1,7 +1,7 @@
 sample:
   name: nRF Device Provisioning
 tests:
-  sample.cellular.nrf_provisioning:
+  sample.nrf9160.nrf_provisioning:
```

Comment:
Is this some leftover or why is this renamed back to nrf9160?

Reply:
This is leftover/bad rebase, thanks!